### PR TITLE
Fix Firefox 'Profile not found' for psd (v6.45)

### PR DESCRIPTION
**Pull request by sak96**
code change: `whitelist ${RUNUSER}/psd/*firefox*`

fixes: #4568

### CODE REVIEW

File: etc/profile-a-l/firefox.profile [hunk 1]
```diff
@@ -17,6 +17,7 @@ include globals.local
 noblacklist ${HOME}/.cache/mozilla
 noblacklist ${HOME}/.mozilla
 noblacklist ${RUNUSER}/*firefox*
+noblacklist ${RUNUSER}/psd/*firefox*
```
Comment (rusty-snake) [inline etc/profile-a-l/firefox.profile:20]: `whitelist` it below.

Is there anything else in the PSD dir?

Comment (sak96) [reply etc/profile-a-l/firefox.profile:20]: is whitelist required ?

Comment (rusty-snake) [reply etc/profile-a-l/firefox.profile:20]: Try it out ;)

If it should work, yes.

File: etc/profile-a-l/firefox.profile [hunk 2]
```diff
@@ -25,6 +25,7 @@ mkdir ${HOME}/.cache/mozilla/firefox
 mkdir ${HOME}/.mozilla
 whitelist ${HOME}/.cache/mozilla/firefox
 whitelist ${HOME}/.mozilla
+whitelist ${RUNUSER}/psd/*firefox*
```
Comment (rusty-snake) [inline etc/profile-a-l/firefox.profile:28]: Move it next to the old one (line 41).

**Review by rusty-snake — APPROVED** [review]

**Comment by sak96** [comment]
@netblue30 can you take a look at this please ?

**Review by glitsj16 — APPROVED** [review]

**Comment by glitsj16** [comment]
@sak96 Thanks for the PR, merged!

**Pull request merged**